README.md

```diff
@@ -3,9 +3,11 @@ Dial a Christmas Carol - for Advent Calendar
 
 You need to have the following installed:
 
-* Python 3.x
-* Request
+[Python 3.x](https://www.python.org/downloads/)
+* Request 
+-Use the following command in your terminal to install Request: pip install requests 
```

Comment:
We use backticks to highlight commands that people should execute, for example `pip install requests`, please can you update to include these?